src/Symfony/Bundle/FrameworkBundle/Resources/config/schema/symfony-1.0.xsd

```diff
@@ -10,7 +10,6 @@
     <xsd:simpleType name="validator_api_version">
```

Comment:
I don't think we want multiple validator apis in 3.0. Also you removed the api configuration as well: https://github.com/symfony/symfony/pull/15683/files#diff-5e7347edce37c5886ec67b7ba02f3a9cL475